examples/random_forest/job_configs/random_forest_base/meta.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "xgboost_tree_bagging",
```

Comment:
follow my other comments, can we move this into

"/examples/xgboost/tree-based"?